py3-ansible.yaml

```diff
@@ -0,0 +1,75 @@
+package:
+  name: py3-ansible
+  version: 2.18.0
+  epoch: 0
+  description: Ansible is a radically simple IT automation platform
+  copyright:
+    - license: GPL-3.0
+  dependencies:
+    provider-priority: 0
+    runtime:
+      - py3-jinja2
+      - py3-yaml
+
+vars:
+  pypi-package: ansible
+  import: ansible
+
+data:
+  - name: py-versions
+    items:
+      3.10: "310"
+      3.11: "311"
+      3.12: "312"
+      3.13: "300"
+
+environment:
+  contents:
+    packages:
+      - py3-supported-build-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: ec78526b976481f3fcdf91a904eeaa68b89ccdea
+      repository: https://github.com/ansible/ansible
+      tag: v${{package.version}}
+
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: python${{range.key}} version of ${{vars.pypi-package}}
+    dependencies:
+      provider-priority: ${{range.value}}
+      provides:
+        - py3-${{vars.pypi-package}}
+    pipeline:
+      - name: Python Build
+        uses: python/build-wheel
+      - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: ansible/ansible
+    strip-prefix: v
+
+test:
```

Comment:
We should add an import test as well. See other versioned Python packages as examples